Cargo.toml

```diff
@@ -34,6 +34,7 @@ byteorder = "1"
 flatgeobuf = { version = "3.27.0", optional = true, default-features = false }
 gdal = { version = "0.15", optional = true }
 geo = "0.26"
+geo-types = "0.7.11"
```

Comment:
geo-types is re-exported through geo, so we shouldn't have a dependency on both of them